scripts/install_third_party_libs.py

```diff
@@ -123,8 +123,8 @@ def pip_install(package, version, install_path):
         raise Exception
 
     subprocess.check_call([
-        'pip', 'install', '%s==%s' % (package, version), '--target',
-        install_path])
+        'python', '-m', 'pip', 'install', '%s==%s' % (package, version),
```

Comment:
Also, what's the reason for this change?

Reply:
See comment above for the reason. It didn't fix or change anything for me, but this is more to smooth things out if things were to go wrong.

Reply:
Hm, ok -- can you add a comment to the code too? Seems subtle.

Reply:
sure, done!